src/sdk/nb.d.ts

```diff
@@ -403,6 +403,8 @@ interface ObjectInfo {
     obj_id: string;
     bucket: string;
     key: string;
+    object_owner_id: string,
```

Comment:
can you change it to `owner: object` ? and return id + name together?

Reply:
sure. incorporated the changes